shared/utilities/ticket-status/index.js

```diff
@@ -64,8 +64,8 @@ export function getTicketStatusSubtitle(event, ticketStatusOptions) {
 }
 
 export function getTicketStatusOptions(event) {
-  const ticketStatusOptions = statusTypes[event.status];
-  if (ticketStatusOptions) {
+  if (statusTypes[event.status]) {
+    const ticketStatusOptions = Object.assign({}, statusTypes[event.status]);
```

Comment:
@asavin You could destructure instead?

    const ticketStatusOptions = { ...statusTypes[event.status] };